src/Cmixin/Holidays/de-national.php

```diff
@@ -2,31 +2,14 @@
 
 return [
     'new-year'                => '01-01',
-    '02-14'                   => '02-14',
-    'easter-52'               => '= easter -52',
-    'easter-48'               => '= easter -48',
-    'easter-46'               => '= easter -46',
-    'easter-3'                => '= easter -3',
     'easter-2'                => '= easter -2',
     'easter'                  => '= easter',
     'easter-p1'               => '= easter 1',
     '05-01'                   => '05-01',
-    '2nd-sunday-of-may'       => '= second Sunday of May',
     'easter-39'               => '= easter 39',
-    'easter-49'               => '= easter 49',
     'easter-50'               => '= easter 50',
     '10-03'                   => '10-03',
-    '11-01'                   => '11-01',
-    '11-02'                   => '11-02',
     '11-11'                   => '11-11',
```

Comment:
I can't find 11/11 in the Wikipedia list. I will remove it.

Reply:
Oh, that one I missed. 11/11 is again a carnival's day, so it's not an official holiday in any German state.